common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/LiferayCore.java

```diff
@@ -48,6 +61,62 @@
 
     private static LiferayProjectProviderReader providerReader;
 
+    private void checkMultipleSDKMarker( final IProgressMonitor monitor ) throws CoreException
```

Comment:
Hi greg, I have to add this method in here because this method will be invoked while the ide started. The liferay core plugin can't use the sdkUtil, it will cause dependencies cycle.